hera_pspec/uvpspec_utils.py

```diff
@@ -366,3 +364,37 @@ def _conj_blpair(blpair, which='both'):
 
     return conj_blpair
 
+def _fast_lookup_blpairts(src_blpts, query_blpts):
+    """
+    Helper function to allow fast lookups of array indices for large arrays of 
+    blpair-time tuples.
+    
+    Parameters
+    ----------
+    src_blpts : array_like
```

Comment:
This says `array_like`, but the description makes it seem like what you actually want is a list, something like
```
src_blpts = [(blp1, time1), (blp2, time2), ...]
```
in which case the data type should be `list` not `array_like`